### ISSUE TITLE
differnet loss functions

### ISSUE THREAD

**Issue by jaxels20** [rnygar20/P7#13]

**Comment by jaxels20** [comment]
We want to use three different loss functions: 
Negative Log likelihood
Focal loss
Cross Entropy
